src/Pokemon.jsx

```diff
@@ -25,38 +27,48 @@ const Pokemon = (props) => {
     const { name, id, species, height, weight, types, sprites } = pokemon;
     const fullImageUrl = `https://pokeres.bastionbot.org/images/pokemon/${id}.png`;
     const { front_default } = sprites;
-    const typeJSX = types.map((typeInfo) => {
+
+    const pokemonTypes = types.map((typeInfo) => {
       const { type } = typeInfo;
       const { name } = type;
-      return <Typography key={name}> {`${name}`}</Typography>;
+      return (
+        <Typography key={toFirstCharUppercase(name)}>
+          {' '}
+          {`${toFirstCharUppercase(name)}`}
+        </Typography>
+      );
     });
     return (
       <>
         <Typography variant="h1">
           {`${id}.`} {toFirstCharUppercase(name)}
-          <img src={front_default} />
+          <img src={front_default} alt="pokemon img" />
         </Typography>
-        <img style={{ width: '300px', height: '300px' }} src={fullImageUrl} />
+        <img
+          style={{ width: '300px', height: '300px' }}
+          src={fullImageUrl}
+          alt="pokemon img"
```

Comment:
You had warnings that your images did not have `alt` tags. You always need alt tags on an image for blind users.